.circleci/config.yml

```diff
@@ -189,19 +189,17 @@ jobs:
             done
       - docker/install-docker-credential-helper
       - docker/configure-docker-credentials-store
-      - run:
-          name: Build Docker image(s)
-          command: |
-            make build-docker
-            for tag in "${DOCKER_IMAGE_TAGS[@]}"; do
-              docker tag offen/offen:local offen/offen:$tag
-            done
       - run:
           name: Push offen/offen to Docker Hub
           command: |
             echo "$DOCKER_ACCESSTOKEN" | docker login --username $DOCKER_LOGIN --password-stdin
+            # This is required for building ARM: https://gitlab.alpinelinux.org/alpine/aports/-/issues/12406
```

Comment:
There's one more thing I just noticed. We'll still need to make sure CircleCI uses a Docker version that supports `buildx` and also set some more env vars to enable it. See: https://github.com/offen/docker-volume-backup/blob/30265c14ba967219e7c2dd187b8d71396ea5a5ca/.circleci/config.yml#L27-L34 for reference.

Reply:
If this is too confusing for you to get done (I work with CircleCI config _a lot_ so I might be blind), I'm also happy to add it. Just let me know.

Reply:
CircleCI is new to new, but yml isn't. I've added the flags for buildx